lib/file.js

```diff
@@ -134,17 +142,15 @@ async function downloadFileConcurrently(url, filepath, options) {
     // extract upload options
     const downloadOptions = filterObject(
         options || {},
-        [ 'timeout', 'headers', 'maxConcurrent', 'concurrent',
-            'retryMaxDuration', 'retryBackoff', 'retryInitialDelay', 'retryEnabled',
-            'retryAllErrors', 'retryMaxCount', 'preferredPartSize' ]
+        ['timeout', 'headers', ...CONCURRENCY_OPTIONS_FILTER, ...RETRY_OPTIONS_FILTER, 'preferredPartSize']
```

Comment:
timeout is a retry option
(`timeout` or `socketTimeout`)

Reply:
ACK, will move it

Reply:
(will be in next refactoring PR)